api/StorageManager.json

```diff
@@ -5,34 +5,56 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/StorageManager",
         "support": {
           "webview_android": {
-            "version_added": null
+            "version_added": "48"
           },
           "chrome": {
-            "version_added": null
+            "version_added": "48"
           },
           "chrome_android": {
-            "version_added": null
+            "version_added": "48"
           },
           "edge": {
             "version_added": null
           },
           "edge_mobile": {
             "version_added": null
           },
-          "firefox": {
-            "version_added": null
-          },
+          "firefox": [
+            {
+              "version_added": "57"
+            },
+            {
+              "version_added": "51",
+              "version_removed": "57",
+              "notes": "See <a href='https://bugzil.la/1304966'>Bug 1304966</a> and <a href='https://bugzil.la/1399038'>Bug 1399038</a>.",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.storageManager.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            }
+          ],
           "firefox_android": {
-            "version_added": null
+            "version_added": "51",
+            "notes": "See <a href='https://bugzil.la/1304966'>Bug 1304966</a> and <a href='https://bugzil.la/1399038'>Bug 1399038</a>.",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "dom.storageManager.enabled",
+                "value_to_set": "true"
+              }
+            ]
           },
           "ie": {
             "version_added": null
           },
           "opera": {
-            "version_added": null
+            "version_added": true
```

Comment:
Since I updated the Chrome versions and am not aware of a Chrome version to Opera version mapping (like [the one for Samsung Internet]), I have reset these versions to `true`.

Reply:
Opera is usually Chrome - 13, so in this case 35. But I've always been a little bit uncomfortable doing that on an automatic basis, so you don't actually have to change this. I'll leave it up to your judgment